google/cloud/spanner/internal/session_pool.cc

```diff
@@ -197,6 +204,48 @@ void SessionPool::Erase(std::string const& session_name) {
   }
 }
 
+Status SessionPool::CreateMultiplexedSession() {
+  std::unique_lock<std::mutex> lk(mu_);
+  if (!multiplexed_session_ || multiplexed_session_->is_bad()) {
+    auto stub = GetStub(std::move(lk));
+    auto name = CreateMultiplexedSession(std::move(stub));
```

Comment:
I assume it is safe to call `CreateMultiplexedSession()` without a lock because it is not changing the state of `*this`?  Should it be defined as `const` then?

Reply:
Done.